### ISSUE TITLE
TypeError: Cannot read property 'getGroups' of undefined

### ISSUE THREAD

**Issue by Nicolapps** [skills17/karma-helpers#1]
When some test fails because of a syntax error or a timeout, the error message that we get isn’t helpful to find the source of the issue. This is what’s currently shown:

```
> skills17-bugreport@1.0.0 test {project path}
> karma start


------------       RESULT       ------------

Summary:
28 01 2021 21:19:39.815:ERROR [karma-server]: UncaughtException: TypeError: Cannot read property 'getGroups' of undefined
    at Printer.print ({project path}/node_modules/@skills17/test-result-printer/lib/Printer.js:33:14)
    at Skills17Reporter.onBrowserComplete ({project path}/node_modules/@skills17/karma-helpers/lib/reporter.js:41:15)
    at Server.<anonymous> ({project path}/node_modules/karma/lib/events.js:40:26)
    at Server.emit (events.js:327:22)
    at Browser.onComplete ({project path}/node_modules/karma/lib/browser.js:94:20)
    at Socket.<anonymous> ({project path}/node_modules/karma/lib/browser.js:216:44)
    at Socket.emit (events.js:315:20)
    at {project path}/node_modules/socket.io/dist/socket.js:434:28
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
28 01 2021 21:19:39.816:ERROR [karma-server]: TypeError: Cannot read property 'getGroups' of undefined
    at Printer.print ({project path}/node_modules/@skills17/test-result-printer/lib/Printer.js:33:14)
    at Skills17Reporter.onBrowserComplete ({project path}/node_modules/@skills17/karma-helpers/lib/reporter.js:41:15)
    at Server.<anonymous> ({project path}/node_modules/karma/lib/events.js:40:26)
    at Server.emit (events.js:327:22)
    at Browser.onComplete ({project path}/node_modules/karma/lib/browser.js:94:20)
    at Socket.<anonymous> ({project path}/node_modules/karma/lib/browser.js:216:44)
    at Socket.emit (events.js:315:20)
    at {project path}/node_modules/socket.io/dist/socket.js:434:28
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
28 01 2021 21:19:39.822:WARN [Chrome Headless 88.0.4324.96 (Mac OS 10.15.7)]: Disconnected (0 times) Client disconnected from CONNECTED state (transport close)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! skills17-bugreport@1.0.0 test: `karma start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the skills17-bugreport@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by cyrilwanner** [comment]
Thank you for the bug report! That should definitely not happen, I'll take a look at it 👍

**Comment by cyrilwanner** [comment]
It should now be fixed in `v1.0.2`.
When a syntax error occurs, no test will be executed at all so we are not able to display the result section.
But the actual syntax error (or any other browser error) is now displayed correctly in the console.
![image](https://user-images.githubusercontent.com/2709252/106383349-ad2c4980-63c5-11eb-9238-1e9017fbb0c1.png)

**Comment by Nicolapps** [comment]
Perfect, thank you for the fix! 😀
